### PR TITLE
DOC: Add conda uninstall pandas to contributing guide

### LINKED ISSUE #25487: DOC: Force Removal of pandas From Dependencies
From a discussion on Gitter with @TomAugspurger when following the pandas contributing guide the step:

```sh
conda env create -f environment.yml
```

Causes pandas to be installed in the environment from conda as a result of dependency resolution with third party libraries (ex: statsmodels, pyarrow, seaborn). 

To prevent undesired conflicts between the local development version and the installed version we should update the documentation to have an explicit removal of the bundled version as such:

```sh
conda uninstall --force pandas
```

### PR DESCRIPTION
**Pull request by yehia67**
closes https://github.com/pandas-dev/pandas/issues/25487 
add command 
`conda uninstall --force pandas`

### CODE REVIEW

**Review by WillAyd — APPROVED** [review]
lgtm @TomAugspurger

File: doc/source/development/contributing.rst [hunk 1]
```diff
@@ -177,6 +177,7 @@ We'll now kick off a three-step process:
 
    # Create and activate the build environment
    conda env create -f environment.yml
+   conda uninstall --force pandas
```
Comment (TomAugspurger) [inline doc/source/development/contributing.rst]: You should activate before uninstalling. So swap this line and the next.

**Review by jreback — CHANGES_REQUESTED** [review]
per @TomAugspurger comment

**Comment by TomAugspurger** [comment]
Thanks @yehia67!

**Pull request merged**